### PR TITLE
Export: remove old export functionality

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

File: js/ui/data_grid/ui.data_grid.export.js [hunk 1]
```diff
@@ -622,38 +600,24 @@ export const ExportController = dataGridCore.ViewController.inherit({}).inherit(
...
     exportTo: function(selectedRowsOnly, format) {
         this._selectionOnly = selectedRowsOnly;
 
         const onExporting = this.getAction('onExporting');
         const eventArgs = {
+            rtlEnabled: this.option('rtlEnabled'),
             selectedRowsOnly: !!selectedRowsOnly,
             format,
-            fileName: this.option('export.fileName'),
+            fileName: 'DataGrid',
```
Comment (pomahtri) [inline js/ui/data_grid/ui.data_grid.export.js:610]: Why we need to pass this field?

File: js/ui/data_grid/ui.data_grid.export.js [hunk 2]
```diff
@@ -781,13 +743,12 @@ dataGridCore.registerModule('export', {
                     const items = [];
 
                     formats.forEach((formatType) => {
-                        let exportMethod = 'exportTo';
+                        const exportMethod = 'exportTo';
```
Comment (pomahtri) [inline js/ui/data_grid/ui.data_grid.export.js]: let's remove this and change

```
this._exportController[exportMethod]
```

to 

```
this._exportController.exportTo
```

below in code

Comment (EugeniyKiyashko) [reply js/ui/data_grid/ui.data_grid.export.js]: [fixed](https://github.com/DevExpress/DevExtreme/pull/23643/commits/318a8ef73a88264d78c65dbe2c25a6e7c5ea541d)

File: js/ui/data_grid/ui.data_grid.export.js (hunk 1 above)
Comment (EugeniyKiyashko) [reply js/ui/data_grid/ui.data_grid.export.js:610]: Yes it is documented field

File: js/ui/data_grid/ui.data_grid.export.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { extend } from '../../core/utils/extend';
 import { getDefaultAlignment } from '../../core/utils/position';
 import dataGridCore from './ui.data_grid.core';
 import { prepareItems } from '../grid_core/ui.grid_core.export';
-import { export as clientExport, excel } from '../../exporter';
+// import { export as clientExport, excel } from '../../exporter';
```
Comment (ksercs) [inline js/ui/data_grid/ui.data_grid.export.js:8]:
```suggestion
```

**Review by ksercs — APPROVED** [review]

File: js/ui/data_grid/ui.data_grid.export.js (hunk 3 above)
Comment (EugeniyKiyashko) [reply js/ui/data_grid/ui.data_grid.export.js:8]: Fixed already https://github.com/DevExpress/DevExtreme/pull/23643/commits/12251f0b8f6aeeb867c9347bf1f0b3be1149a22d

**Review by andrewmakarov — APPROVED** [review]

**Review by pomahtri — APPROVED** [review]

**Review by andrewmakarov — APPROVED** [review]

**Pull request merged**